tests/components/bmw_connected_drive/test_diagnostics.py

```diff
@@ -22,9 +21,16 @@
 
 @pytest.mark.freeze_time(datetime.datetime(2022, 7, 10, 11))
 async def test_config_entry_diagnostics(
-    hass: HomeAssistant, hass_client: ClientSessionGenerator, bmw_fixture
+    hass: HomeAssistant,
+    hass_client: ClientSessionGenerator,
+    bmw_fixture,
+    snapshot: SnapshotAssertion,
 ) -> None:
-    """Test config entry diagnostics."""
+    """Test config entry diagnostics.
+
+    Could fail after every change (either to HA or the library). If so, regenerate using
+    pytest --snapshot-update tests/components/bmw_connected_drive/ and check the output.
```

Comment:
I don't think this is the right place to document how snapshotting works 😉

Reply:
Removed. You're right, it was rather a convenience feature for myself to have it stored somewhere quickly available.